grayskull/pypi/pypi.py

```diff
@@ -552,10 +618,36 @@ def _get_metadata(self) -> dict:
         version = ""
         if self["package"]["version"].values:
             version = self.get_var_content(self["package"]["version"].values[0])
-        pypi_metadata = self._get_pypi_metadata(name, version)
-        sdist_metadata = self._get_sdist_metadata(
-            sdist_url=pypi_metadata["sdist_url"], name=name
-        )
+        if origin_is_github(name):
+            # TODO: Clean this function up a bit.
+            url = name
+            name = name.split("/")[-1]
+            if version:
+                # try get the tag with the most similar name to the requested version
+                version_tag = self._get_most_similar_tag_in_repo(url, version)
+                log.info(f"Closest git reference to `{version}` is `{version_tag}`.")
+            else:
+                version_tag = self._get_latest_version_of_github_repo(url)
+                log.info(
+                    f"Version for {name} not specified."
+                    + "\nGetting the latest one, which is {version_tag}."
```

Comment:
You don't need `+` thanks to "implicit string concatenation"

```suggestion
                    "\nGetting the latest one, which is {version_tag}."
```